JPS_Ontology/ontology/ontomops/ontomops.csv

```diff
@@ -0,0 +1,57 @@
+isFunctioningAs,Object Property,,,ChemicalBuildingUnit,GenericBuildingUnit,only,A relation that connects the chemical building unit and the more abstract generic building unit. ,https://www.theworldavatar.com/kg/ontomops/,
+http://www.theworldavatar.com/ontology/ontospecies/OntoSpecies.owl#hasMolecularWeight,Object Property,,,MetalOrganicPolyhedron UNION ChemicalBuildingUnit,http://www.theworldavatar.com/ontology/ontospecies/OntoSpecies.owl#MolecularWeight,,A relation between a molecular entity and its molecular weight.,https://www.theworldavatar.com/kg/ontomops/,
+http://www.ontology-of-units-of-measure.org/resource/om-2/hasValue,Object Property,,,http://www.theworldavatar.com/ontology/ontospecies/OntoSpecies.owl#Charge,http://www.ontology-of-units-of-measure.org/resource/om-2/Measure,,,http://www.ontology-of-units-of-measure.org/resource/om-2/,
+http://www.ontology-of-units-of-measure.org/resource/om-2/hasUnit,Object Property,,,http://www.ontology-of-units-of-measure.org/resource/om-2/Measure,http://www.ontology-of-units-of-measure.org/resource/om-2/Unit,,,http://www.ontology-of-units-of-measure.org/resource/om-2/,
+http://www.ontology-of-units-of-measure.org/resource/om-2/hasValue,Object Property,,,http://www.theworldavatar.com/ontology/ontospecies/OntoSpecies.owl#MolecularWeight,http://www.ontology-of-units-of-measure.org/resource/om-2/Measure,,,http://www.ontology-of-units-of-measure.org/resource/om-2/,
+http://www.ontology-of-units-of-measure.org/resource/om-2/hasValue,Object Property,,,Volume,http://www.ontology-of-units-of-measure.org/resource/om-2/Measure,,,http://www.ontology-of-units-of-measure.org/resource/om-2/,
+hasCBUFormula,Data Property,,,ChemicalBuildingUnit,String,,Data string outlying the formula of the chemical building unit. ,https://www.theworldavatar.com/kg/ontomops/,
+hasMOPFormula,Data Property,,,MetalOrganicPolyhedron,String,,,https://www.theworldavatar.com/kg/ontomops/,
+hasCCDCNumber,Data Property,,,MetalOrganicPolyhedron,String,,A data property that informs on the exact deposition number of the MOP in the CCDC database.,https://www.theworldavatar.com/kg/ontomops/,
+http://www.theworldavatar.com/ontology/ontospecies/OntoSpecies.owl#Geometry,Class,,,,,,,http://www.theworldavatar.com/ontology/ontospecies/OntoSpecies.owl,
+http://www.theworldavatar.com/ontology/ontospecies/OntoSpecies.owl#hasGeometry,Object Property,,,MetalOrganicPolyhedron UNION ChemicalBuildingUnit,http://www.theworldavatar.com/ontology/ontospecies/OntoSpecies.owl#Geometry,,A relation between a molecular entity and the geometry associated with it. ,https://www.theworldavatar.com/kg/ontomops/,
+hasGeometryFile,Data Property,,,http://www.theworldavatar.com/ontology/ontospecies/OntoSpecies.owl#Geometry,String,,The path to the XYZ file representing a molecular geometry.,https://www.theworldavatar.com/kg/ontomops/,
```

Comment:
It would be better to define data property `hasGeometryFile` under the namespace `OntoSpecies` as here it's essentially defining data property for a class belonging to `OntoSpecies`.

Reply:
Agreed, moved it.
Commit 00268447a9ee1888e18ee7ad9c83cdde92a3c587